packages/create-astro/src/index.ts

```diff
@@ -233,30 +211,40 @@ export async function main() {
 		initial: true,
 	});
 
-	if (!gitResponse) {
-		process.exit(0);
-	}
-
-	if (gitResponse.git && !args.dryrun) {
+	if (args.dryRun) {
+		ora().info(dim(`--dry-run enabled, skipping.`));
+	} else if (gitResponse.git) {
 		await execaCommand('git init', { cwd });
+	} else {
+		ora().info(
+			dim(`Sounds good! You can come back and run ${cyan(`git init`)} in your project anytime.`)
+		);
 	}
 
-	ora({ text: green('Done. Ready for liftoff!') }).succeed();
+	ora().succeed('Setup complete.');
+	ora({ text: green('Ready for liftoff!') }).succeed();
+	await wait(300);
+
 	console.log(`\n${bgCyan(black(' Next steps '))}\n`);
 
-	const projectDir = path.relative(process.cwd(), cwd);
+	let projectDir = path.relative(process.cwd(), cwd);
 	const devCmd = pkgManager === 'npm' ? 'npm run dev' : `${pkgManager} dev`;
 
-	console.log(
+	await logAndWait(
 		`You can now ${bold(cyan('cd'))} into the ${bold(cyan(projectDir))} project directory.`
 	);
-	console.log(
+	await logAndWait(
 		`Run ${bold(cyan(devCmd))} to start the Astro dev server. ${bold(cyan('CTRL-C'))} to close.`
 	);
-	if (!installResponse.install) {
-		console.log(yellow(`Remember to install dependencies first!`));
-	}
-	console.log(`\nStuck? Come join us at ${bold(cyan('https://astro.build/chat'))}`);
+	await logAndWait(
+		`Add frameworks like ${bold(cyan('react'))} and ${bold(
+			cyan('tailwind')
+		)} to your project using ${bold(cyan('astro add'))}`
+	);
+	await logAndWait('');
+	await logAndWait(`Stuck? Come join us at ${bold(cyan('https://astro.build/chat'))}`, 1000);
+	await logAndWait(dim('Good luck out there, astronaut.'));
+	await logAndWait('', 300);
```

Comment:
Nice touch with the staggered output!

Reply:
Thanks! I was kind of just playing around at first but it feels a lot more human.

The `1000` wait before "good luck out there" is a bit of a cheesy touch, so if anyone has any thoughts I'm okay to scrap it or shrink it down to a smaller wait. You can see it in action in the video above.